cle/backends/binja.py

```diff
@@ -1,20 +1,25 @@
 import logging
 
-from . import Backend, register_backend, Symbol, SymbolType
-from .relocation import Relocation
-from ..errors import CLEError
-from ..address_translator import AT
 import archinfo
 
-l = logging.getLogger(name=__name__)
+from cle.address_translator import AT
+from cle.errors import CLEError
+
+from .backend import Backend, register_backend
+from .relocation import Relocation
+from .symbol import Symbol, SymbolType
+
+log = logging.getLogger(name=__name__)
```

Comment:
can we please disable whatever lint check encouraged you to name loggers this way, or maybe add an exception for the name `l`?

Reply:
Thought about it some more. I will downgrade this from a warning to a weak preference if we can add this commit to the ignore-revs file.

Reply:
I looked and I could disable the check, but there didn't seem to be a way to explicitly allow `l`. This commit will definitely go into the ignore revs file.